servlet/src/main/java/org/togglz/servlet/activation/ClientIpActivationStrategy.java

```diff
@@ -49,7 +49,10 @@ public boolean isActive(FeatureState featureState, FeatureUser user)
          List<String> parts = Strings.splitAndTrim(featureState.getParameter(PARAM_IPS), "[\\s,]+");
 
          try {
-            String remoteAddr = request.getRemoteAddr();
+             String remoteAddr = request.getHeader("X-FORWARDED-FOR");
```

Comment:
Could you use `X-Forwarded-For` instead of all uppercase. That's how the header is typically sent by the client.